index.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="main.css">
+</head>
+<body>
+  <header id="top">
+    <a href="#"><img src="./pics/LogoPlaceholder.png" alt="logo"></a>
+    <nav>
+      <a href="#"><img src="./pics/Icon-Menu.png" alt="menu"></a>
+      <ul>
+      <li><a href="#">Portfolio</a></li>
+      <li><a href="#">About</a></li>
+      <li><a href="#">Contact</a></li>
+    </ul>
+    </nav>
+  </header>
+  <main>
+    <section id="headline">
+      <h1>Hey there. I'm Edozie. <br> <span id="title-span">I'm a software developer</span></h1>
+      <p class="text">
+        I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <ul class="socials">
+        <li><a href="#"><img src="./pics/github.png" alt="github logo"></a></li>
+        <li><a href="#"><img src="./pics/linkedin.png" alt="linkedin logo"></a></li>
+        <li><a href="#"><img src="./pics/Enabled.png" alt="enabled"></a></li>
+        <li><a href="#"><img src="./pics/twitter.png" alt="twitter"></a></li>
+        <li><a href="#"><img src="./pics/medium.png" alt="medium"></a></li>
+      </ul>
+      <div >
+        <a href="#"><img src="./pics/Action.png" alt="action button"></a>
+      </div>
+    </section>
+    <section id="works">
+      <header id="recent">
+        <h1>My Recent Works</h1>
+        <div id="indicator"></div>
+      </header>
+
+      <ul class="work-row">
+        <li class="item">
+          <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+          <div class="text-card">
+            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <ul class="courses">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          <button type="submit">See Project</button>
+          </div>
+        </li>
```

Comment:
Kindly notice the differences in the font size and weight for all the work cards and fix them 🤗
  - _The font size of the headings of the projects should be bigger than the following text in the section._

Your design:

![image](https://user-images.githubusercontent.com/73714615/187035775-3d275a60-e89e-4e3a-ba69-497f979a9ac2.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187035788-3b2848a5-1da8-4846-944c-a96c818609b0.png)